src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -664,8 +630,68 @@ public void createGeneratorWithStartup() throws Exception {
     }
 
     @Test
-    public void testMoveModificationWithoutOrigin() throws Exception {
-        testMoveModification(null, null);
+    public void testMoveModificationInSameGroup() throws Exception {
```

Comment:
Is there a way to test that modifications are applied correctly on the network afterward ? Because I think this part of the code is not tested for now